De-emphasize the -h option and refer people to systemd instead

---
 man/rpcbind.8 |   34 ++++++++++++++--------------------
 1 file changed, 14 insertions(+), 20 deletions(-)

Index: rpcbind-0.2.1_rc4/man/rpcbind.8
===================================================================
--- rpcbind-0.2.1_rc4.orig/man/rpcbind.8
+++ rpcbind-0.2.1_rc4/man/rpcbind.8
@@ -86,27 +86,21 @@ checks are shown in detail.
 Do not fork and become a background process.
 .It Fl h
 Specify specific IP addresses to bind to for UDP requests.
-This option
-may be specified multiple times and is typically necessary when running
-on a multi-homed host.
-If no
-.Fl h
-option is specified,
-.Nm
-will bind to
-.Dv INADDR_ANY ,
-which could lead to problems on a multi-homed host due to
+This option may be specified multiple times and can be used to
+restrict the interfaces rpcbind will respond to.
+Note that when
 .Nm
-returning a UDP packet from a different IP address than it was
-sent to.
-Note that when specifying IP addresses with
-.Fl h ,
-.Nm
-will automatically add
-.Li 127.0.0.1
-and if IPv6 is enabled,
-.Li ::1
-to the list.
+is controlled via systemd's socket activation,
+the
+.Fl h
+option is ignored. In this case, you need to edit
+the
+.Nm ListenStream
+and
+.Nm ListenDgram
+definitions in
+.Nm /usr/lib/systemd/system/rpcbind.socket
+instead.
 .It Fl i
 .Dq Insecure
 mode.
